src/components/Loader/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,85 @@
+import { useEffect, useState } from "react"
+import { Person } from "../../../types";
+import { getPeople } from "../../../api";
+import { PersonItem } from "../PersonItem/PersonItem";
+import { useParams } from "react-router-dom";
+import { Loader } from "../Loader/Loader";
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const { personSlug } = useParams();
+
+  useEffect(() => {
+    setIsLoading(true);
+    getPeople().then(setPeople)
+    .catch(() => {
+      setError('Something went wrong')
+    })
+    .finally(() => {
+      setIsLoading(false)
+    })
+
+  }, [])
+
+  const getFather = (name: string | null) => people.find(person => person.name === name);
+
+  const getMother = (name: string | null) => people.find(person => person.name === name);
```

Comment:
You can use function(ex., `getPreparedPeople()`) to add `mother` and `father` properties to `person` object. Something like that:

```
  preparedPeople.map(person => {
    const father = preparedPeople.find(
      parent => parent.name === person.fatherName,
    );
    const mother = preparedPeople.find(
      parent => parent.name === person.motherName,
    );

    return { ...person, father, mother };
  });
```

Reply:
> You can use function(ex., `getPreparedPeople()`) to add `mother` and `father` properties to `person` object. Something like that:
> 
> ```
>   preparedPeople.map(person => {
>     const father = preparedPeople.find(
>       parent => parent.name === person.fatherName,
>     );
>     const mother = preparedPeople.find(
>       parent => parent.name === person.motherName,
>     );
> 
>     return { ...person, father, mother };
>   });
> ```

Good point, create a function and move it to utiles/helpers